tests/framework/utils_vsock.py

```diff
@@ -205,10 +205,8 @@ def _vsock_connect_to_guest(uds_path, port):
 
 def _copy_vsock_data_to_guest(ssh_connection, blob_path, vm_blob_path, vsock_helper):
     # Copy the data file and a vsock helper to the guest.
+
     cmd = "mkdir -p /tmp/vsock"
-    cmd += " && mount -t tmpfs tmpfs -o size={} /tmp/vsock".format(
-        BLOB_SIZE + 1024 * 1024
-    )
```

Comment:
Do you have any context why we had this so far?

Reply:
No, as far as I can see tmpfs is already mounted in the guest with around 128MB

Reply:
Hmm, I cannot see the mounted tmpfs on /tmp inside the guest. Am I missing something?
```
 def _copy_vsock_data_to_guest(ssh_connection, blob_path, vm_blob_path, vsock_helper):
     # Copy the data file and a vsock helper to the guest.
+    cmd = "mount | grep tmpfs"
+    ecode, stdout, stderr = ssh_connection.execute_command(cmd)
+    print(stdout.read())
+    print(stderr.read())
```
```
$ tools/devtool test -- integration_tests/functional/test_vsock.py --capture=tee-sys
integration_tests/functional/test_vsock.py::test_vsock[ubuntu]
devtmpfs on /dev type devtmpfs (rw,relatime,size=119948k,nr_inodes=29987,mode=755)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev)
tmpfs on /run type tmpfs (rw,nosuid,nodev,mode=755)
tmpfs on /run/lock type tmpfs (rw,nosuid,nodev,noexec,relatime,size=5120k)
tmpfs on /sys/fs/cgroup type tmpfs (ro,nosuid,nodev,noexec,mode=755)
```

Reply:
But yeah, as far as I see, having the file on tmpfs or root (/dev/vda) seems not to matter, since we don't have any performance-sensitive tests using it. In my understanding, the benefit of using tmpfs is just fast file access: https://man7.org/linux/man-pages/man5/tmpfs.5.html

Reply:
Oh my bad, I think I looked in the new rootfs... in the old rootfs there seems to be enough free space that it doesn't make a difference, but maybe for the perf tests it will?